frontend/routes/ConfigRoute.tsx

```diff
@@ -1,16 +1,23 @@
 import * as React from 'react';
-import { BreadcrumbActions, GameVisibilityActions } from '../storage/common';
+import axios from 'axios';
+import { BreadcrumbActions, GameVisibilityActions, SERVER_ADRESS, DownloadActions, AppActions, AppStore } from '../storage/common';
 import { connect } from 'react-redux';
 import { Dispatch, AnyAction } from 'redux';
 import { ConfigView } from '../views/ConfigView';
+import { Game } from '../storage/models/Game';
 
 interface ConfigRouteProps {
+	games: Map<number, Game>;
 	dispatch: Dispatch<AnyAction>;
 }
 
 export class ConfigRoute extends React.Component<ConfigRouteProps> {
 	public componentDidMount() {
 		this.props.dispatch(BreadcrumbActions.setBreadcrumb('Config'));
+
+		// if (this.props.games.size === 0) {
+		fetchGameData()(this.props.dispatch);
```

Comment:
if I understand correctly the idea is for now to just always fetch the date when viewing this screen?

Reply:
Yeah, as I explained on Discord, just fetching if list is empty doesn't look like the good idea, especially considering we still have placeholder data from initial state